app/assets/javascripts/visualizations/highvis/disabledVis.coffee

```diff
@@ -39,7 +39,8 @@ $ ->
           <img src='#{window.icons["novis_timeline"]}'>
           <br>
           <br>
-          Either a time field was not found or there were not enough data
+          Either Timestamp field or Number field is missing from project or
+          there is not enough data. (min of 3 points required)
```

Comment:
It would need to be, "there are not at least three data points".  ( Change
" is" to "are")

On Thu, Nov 12, 2015, 9:09 PM Anthony Salani <notifications@github.com>
wrote:

> In app/assets/javascripts/visualizations/highvis/disabledVis.coffee
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_isenseDev_rSENSE_pull_2296-23discussion-5Fr44743202&d=CwMCaQ&c=lqHimbpwJeF7VTDNof4ddnO1LTp7jTaMHJ2UAxZwrfY&r=_08kerRFUAjn0TwdG8acQg&m=ixJoIvY6WuC2tZUX3ln1QrMJpUpW7mcM9iKjGI6Iaaw&s=KbypkjPIy21YqOZWYHOMceSI-cz6BNH3i7hp8IzwOYY&e=>
> :
>
> > @@ -39,7 +39,8 @@ $ ->
> >            <img src='#{window.icons["novis_timeline"]}'>
> >            <br>
> >            <br>
> > -          Either a time field was not found or there were not enough data
> > +          Either Timestamp field or Number field is missing from project or
> > +          there is not enough data. (min of 3 points required)
>
> This just sounds awkward. How about:
> "Either a timestamp or number field is missing, or there is not at least 3
> data points."
>
> —
> Reply to this email directly or view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_isenseDev_rSENSE_pull_2296_files-23r44743202&d=CwMCaQ&c=lqHimbpwJeF7VTDNof4ddnO1LTp7jTaMHJ2UAxZwrfY&r=_08kerRFUAjn0TwdG8acQg&m=ixJoIvY6WuC2tZUX3ln1QrMJpUpW7mcM9iKjGI6Iaaw&s=JZbGN8LMK354uP7ImGaVJLnnrjFT84jcyvDx9x0VFwk&e=>
> .
>